tests/testthat/_snaps/groupby.md

```diff
@@ -15,9 +15,7 @@
       │ one ┆ 4.0 │
       │ two ┆ 1.0 │
       └─────┴─────┘
-      groups: [[1]]
-      [1] "foo"
-      
+      groups: foo
```

Comment:
Could you add tests for when there are multiple groups?

Reply:
done